utils/build/docker/dotnet/parametric/Endpoints/ApmTestApi.cs

```diff
@@ -266,6 +268,19 @@ private static async Task FinishSpan(HttpRequest request)
         span.Finish();
     }
 
+    private static string Crash(HttpRequest request)
+    {
+        var thread = new Thread(() =>
+        {
+            throw new BadImageFormatException("Expected");
```

Comment:
Just curious, why throw the exception from another thread?

Reply:
If I throw it from the current thread, aspnetcore will catch it and it won't crash the app